### PR TITLE
add org.gnome.Gnote

### PR DESCRIPTION
**Pull request by nedrichards**

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build org.gnome.Gnote

**Comment by nedrichards** [comment]
bot, build org.gnome.Gnote

File: org.gnome.Gnote.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "gnote",
+    "rename-appdata-file": "gnote.appdata.xml",
+    "rename-desktop-file": "gnote.desktop",
+    "rename-icon": "gnote",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
```
Comment (TingPing) [inline org.gnome.Gnote.json]: Probably doesn't use opengl?

**Comment by TingPing** [comment]
Otherwise looks fine. :+1:

**Comment by nedrichards** [comment]
bot, build org.gnome.Gnote

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.gnome.Gnote

**Pull request closed without merging**